lib/ApiRequestor.php

```diff
@@ -129,7 +129,6 @@ private static function _encodeObjects($d)
      */
     public function request($method, $url, $params = null, $headers = null, $apiMode = 'v1', $usage = [])
     {
-        $params = $params ?: [];
```

Comment:
re: your comment from the other PR, do we want to preserve sending of empty bodies? I'm confused why we've never seen this shape of failure before, did we always pass `[]`?

Reply:
This needs to be "[]" because nulling out is risky.
<img width="194" alt="Screenshot 2024-09-27 at 10 54 41 AM" src="https://github.com/user-attachments/assets/735a787d-2a19-4185-99d1-616f636de96d">

And sending a curl request with `[]` body seems to work for me. 
<img width="786" alt="Screenshot 2024-09-27 at 10 55 20 AM" src="https://github.com/user-attachments/assets/810b44fc-0133-430d-915e-a9f248bf281d">

Reply:
Although I am facing the same issue that you are facing.